triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Converter.scala

```diff
@@ -48,52 +47,37 @@ case class Converter(
 )
 
 // Helper to create identifiers pointing to the DAML.Trigger module
-case class TriggerIds(
-    triggerPackageId: PackageId,
-    triggerModuleName: ModuleName,
-    highlevelModuleName: ModuleName,
-    stdlibPackageId: PackageId,
-    mainPackageId: PackageId) {
-  def getId(n: String): Identifier =
-    Identifier(triggerPackageId, QualifiedName(triggerModuleName, DottedName.assertFromString(n)))
-  def getHighlevelId(n: String): Identifier =
-    Identifier(triggerPackageId, QualifiedName(highlevelModuleName, DottedName.assertFromString(n)))
-}
-
-object TriggerIds {
-  def fromDar(dar: Dar[(PackageId, Package)]): TriggerIds = {
-    val triggerModuleName = DottedName.assertFromString("Daml.Trigger.LowLevel")
-    val highlevelModuleName = DottedName.assertFromString("Daml.Trigger")
-    val triggerPackageId: PackageId = dar.all
-      .find {
-        case (pkgId, pkg) =>
-          pkg.modules.contains(triggerModuleName) &&
-            pkg.modules.contains(highlevelModuleName)
-      }
-      .get
-      ._1
-    val stdlibPackageId =
-      dar.all
-        .find {
-          case (pkgId, pkg) =>
-            pkg.modules.contains(DottedName.assertFromString("DA.Internal.LF"))
-        }
-        .get
-        ._1
-    TriggerIds(
+case class TriggerIds(val triggerPackageId: PackageId) {
+  def damlTrigger(s: String) =
+    Identifier(
       triggerPackageId,
-      triggerModuleName,
-      highlevelModuleName,
-      stdlibPackageId,
-      dar.main._1)
-  }
+      QualifiedName(ModuleName.assertFromString("Daml.Trigger"), DottedName.assertFromString(s)))
+  def damlTriggerLowLevel(s: String) =
+    Identifier(
+      triggerPackageId,
+      QualifiedName(
+        ModuleName.assertFromString("Daml.Trigger.LowLevel"),
+        DottedName.assertFromString(s)))
+  def damlTriggerInternal(s: String) =
+    Identifier(
+      triggerPackageId,
+      QualifiedName(
+        ModuleName.assertFromString("Daml.Trigger.Internal"),
+        DottedName.assertFromString(s)))
 }
 
 case class AnyContractId(templateId: Identifier, contractId: AbsoluteContractId)
 
 class ConverterException(message: String) extends RuntimeException(message)
 
 object Converter {
+  private val DA_INTERNAL_ANY_PKGID =
```

Comment:
These package ids were wrong before. It just doesn’t matter because speedy doesn’t care.